src/styles/blocks/card.scss

```
@@ -0,0 +1,103 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  background-color: #fff;
+
+  &__img {
+    display: block;
+
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    display: flex;
+    align-items: space-between;
+
+    margin-bottom: 4px;
+
+    line-height: 18px;
+    font-weight: 500;
+    font-size: 12px;
+    color: $color-dark;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+
+    font-weight: 400;
+    font-size: 10px;
+    color: $color-gray;
+    line-height: 14px;
+  }
+
+  &__product-rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 24px;
+  }
+
+  &__product-reviews {
+    display: flex;
+    align-items: center;
+
+    min-height: 14px;
+
+    font-weight: 400;
+    font-size: 10px;
+    color: $color-dark;
+  }
+
+  &__price-block {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+
+    margin-bottom: 16px;
+    line-height: 18px;
+  }
+
+  &__price-word {
+    font-weight: 400;
+    font-size: 12px;
+    color: $color-gray;
+  }
+
+  &__price {
+    font-weight: 700;
+    font-size: 16px;
+    color: $color-dark;
+  }
+
+  &__buy-link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    transition: background-color 0.3s, color 0.3s;
+
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    border: 1px solid $color-blue;
+
+    text-decoration: none;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+
+    background-color: $color-blue;
+  }
+
+  &__buy-link:hover {
+    color: $color-blue;
+    background-color: #fff;
+  }
```

It's better to put pseudo-classes inside general selectors (rule №10 in the checklist). Also, check rule №12 and change style properties order using [this](https://codeguide.academy/html-css.html#css-order) example to make your code more readable.
```suggestion
  &__buy-link {
    display: flex;
    align-items: center;
    justify-content: center;
    width: 166px;
    height: 40px;
    
    text-decoration: none;
    color: #fff;
    font-weight: 700;
    font-size: 14px;
    text-transform: uppercase;

    background-color: $color-blue;
    border-radius: 5px;
    border: 1px solid $color-blue;
    
    transition: background-color 0.3s, color 0.3s;
    
    &:hover {
      color: $color-blue;
      background-color: #fff;
  }
 }
```